downstream/modules/automation-hub/proc-pushing-container-images-from-your-local.adoc

```diff
@@ -0,0 +1,60 @@
+
+[id="pushing-container-images-from-your-local]
+
+= Pushing container images from your local
+
+.Procedure
+. From a terminal, log into podman, or any container client currently in use.
++
+----
+> podman pull <__container-name__>
+----
++
+. Once the image is pulled, add tags:
++
+----
+> podman tag <container-name> _<server-address>_/<container-name>:<tag name>
+----
++
+. Sign the image after changes have been made, and push it back up:
++
+----
+> podman push _<server-address>_/<container-name>:<tag name>
+--tls-verify=false --sign-by<reference to the gpg key on your local>
+----
++
+If the image is not signed, it can only be pushed with any current signature
+embedded.
+
+. Push the image without signing it:
++
+----
+> podman push _<server-address>_/<container-name>:<tag name>
+--tls-verify=false
+----
++
+. Navigate to the automation hub and click on *Execution Environments* if that
+window is not open.
+
+. Click the "Refresh" icon to refresh the page to show the new execution
+environment.
+
+. Click on the name of the image.
+
+In the details page, below the image name, will be displayed whether or not the
+image has been signed. In this case, it displays "Unsigned."
+
+To sign the image from automation hub:
+
+. Click on the image name to open the details page.
+
+. Click on the three dots in the upper right hand corner of the details page.
+Three options will be available:
+* Use in Controller
+* Delete
+* Sign
+
+. Click sign from the drop-down menu.
+
+The signing service will sign the image. Once the image is signed, the status
+will change to "signed".
```

Comment:
Suggested change: The signing service signs the image. Once the image is signed, the status changes to "signed".